sdk/src/main/scala/com/horizen/account/receipt/EvmLogUtils.scala

```diff
@@ -64,7 +69,7 @@ object EvmLogUtils extends ScorexSerializer[EvmLog] {
       writer.putBytes(log.topics(i).toBytes)
     }
 
-    val data = log.data
+    val data = if (log.data != null) log.data else Array[Byte](0)
```

Comment:
same here

Reply:
Same comment, data is a public field so it can be changed any time